sources/packages/backend/workflow/BPMN/camunda-8/assessment-gateway.bpmn

```diff
@@ -420,14 +420,6 @@
           <zeebe:output source="=null" target="studentDataTaxReturnIncome" />
           <zeebe:output source="=[]" target="parentSubprocesses" />
           <zeebe:output source="=null" target="calculatedDataParentalAssets" />
```

Comment:
shouldn't `calculatedDataParentalAssets` also be removed?

Reply:
Nice catch. I see that it was set to null during initial efforts of IER 1st big chunk implementation. Will update it.